index.html

```diff
@@ -217,10 +216,9 @@ <h4>JSON-LD Authoring and Processing</h4>
 
 					<p>The publication manifest also has a number of authoring flexibilities and compact authoring
 						expressions. For example, it is not always required that object types be explicitly authored, as
-						these are automatically generated during canonicalization when missing (see <a
-							href="#value-objects"></a> for more information). A <a>canonical representation</a> of the
-						manifest data is defined separately; see the separate section on <a href="#webidl"></a> for
-						further details.</p>
+						these are automatically generated during processing when missing (see <a href="#value-objects"
+						></a> for more information). A <a>internal representation</a> of the manifest data is defined
```

Comment:
This reads right now as,
> A internal 

Sounds better with the right article,

>An internal

Reply:
Thanks, yes, fixed now.